core/src/main/java/com/novoda/noplayer/exoplayer/InfoListener.java

```diff
@@ -1,30 +1,12 @@
 package com.novoda.noplayer.exoplayer;
 
-import com.google.android.exoplayer2.Format;
-import com.google.android.exoplayer2.upstream.DataSpec;
-
-import java.io.IOException;
+import java.util.Map;
 
 /**
  * A listener for debugging information.
  */
 public interface InfoListener {
 
-    void onDroppedFrames(int count, long elapsed);
```

Comment:
The reasoning behind this is:

The forwarders have knowledge of exo-player, the listeners are the ones exposed through `no-player` and are not implementation specific. This allows the client to sign up to one listener that can be used for both implementations.

The methods here now appear in one of the exo-player listeners and get transformed in the `InfoForwarder`.